cobra/core/Model.py

```diff
@@ -16,9 +15,10 @@
 import types
 import optlang
 from sympy.core.singleton import S
-from functools import partial
-from cobra.util import AutoVivification
-from cobra import exceptions, config
+from ..util import AutoVivification
```

Comment:
They appeared in other places in the package, I don't really have a preference (didn't know they were discouraged). I'll change them